src/Values.h

```diff
@@ -205,7 +206,7 @@ namespace vc4c
 		LITERAL, LOCAL,  //also contains labels
 		REGISTER,
 		CONTAINER,
-		UNDEFINED,
+		UNDEFINED,       // XXX CombinedOperation has this type
```

Comment:
CombinedOperations currently need to be handled special, since they don't have 1 output.

Usually, when a CombinedOperation is met, the single instructions must be checked separately. Since the instructions are combined very late (just before register-mapping), most of the compiler does not care for them, though.